src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -340,53 +359,16 @@ describe('ReactDOMComponent', () => {
       expect(stubStyle.color).toEqual('green');
     });
 
-    it('should reject attribute key injection attack on markup', () => {
-      spyOn(console, 'error');
-      for (var i = 0; i < 3; i++) {
-        var container = document.createElement('div');
-        var element = React.createElement(
-          'x-foo-component',
-          {'blah" onclick="beevil" noise="hi': 'selected'},
-          null
-        );
-        ReactDOM.render(element, container);
-      }
-      expectDev(console.error.calls.count()).toBe(1);
-      expectDev(console.error.calls.argsFor(0)[0]).toEqual(
-        'Warning: Invalid attribute name: `blah" onclick="beevil" noise="hi`'
-      );
-    });
-
-    it('should reject attribute key injection attack on update', () => {
```

Comment:
Same here about this probably no longer being applicable now that things are passed as properties instead of attributes.